Formula/driftctl.rb

```diff
@@ -39,6 +39,7 @@ def install
 
   test do
     assert_match "v#{version}", shell_output("#{bin}/driftctl version")
-    assert_match "Invalid AWS Region", shell_output("#{bin}/driftctl --no-version-check scan 2>&1", 1)
+    assert_match "Downloading terraform provider: aws",
```

Comment:
@chenrui333 Hi 👋🏻 sorry to bother you with that, but this not the expected behavior of driftctl when not credentials are found. 

Did you changed something on the build pipeline, like define some `AWS_DEFAULT_REGION` env var or something like that ?
I just wanted to be sure that we do not have introduced any unwanted changed in that version 🙏🏻

Reply:
I dont think we have made any things in the runner, but I am more just updating the tests to get the build passed.